src/css/stage.less

```diff
@@ -45,3 +45,28 @@
     background-color: @silexlightgrey;
   }
 }
+
+#powDiv {
+    position: absolute;
+    width: 150px;
+    text-align: center;
+    background-color: #252525;
+    color: #FFF;
+    width: 200px;
+    height: 46px;
+    border-radius: .15em;
+    border: 1px solid #444;
+    z-index: 11111;
+}
+#leftPowDiv {
+    display: inline;
+}
+#imgLeftPowDiv {
+    width: 70px;
+    height: 38px;
+    margin-top: 5px;
+}
+#rightPowA {
+    display: inline;
+    color: #777
+}
```

Comment:
also, please use the existing less variables for colors, see `src/css/_variables.less`, e.g. `background-color: @silexmediumgrey;`